.env

```diff
@@ -0,0 +1,12 @@
+POSTGRES_DB_FOLDER=<where_create_postgres; exapmle ./postgres>
+POSTGRES_DB_HOST=<db_host>
+POSTGRES_DB_PORT=<db_port>
+POSTGRES_DB_NAME=<db_name>
+POSTGRES_USER_NAME=<db_user>
+POSTGRES_DB_PASSWORD=<db_password>
+HASURA_PORT=<hasura_port>
+HASURA_ADMIN_SECRET=<hasura_password>
+JUNO_SSL_MODE=<ssl_mode>
+JUNO_WORKERS_NUMBER=<amount_of_workers>
+RPC_URL=<gaia_node_http_rpc>
+LCD_URL=<gaia_node_http_lcd>
```

Comment:
LCD isn't a real thing. Let's just call this `CLIENT_URL`.

Reply:
ok, done